### DISCUSSION TITLE
koboldcpp slows down when the program minimized or run on the background.

### DISCUSSION THREAD

**Discussion by meAlphanism** [LostRuins/koboldcpp#1179] [Q&A] [answered]
Hello, everyone. I would like to raise a question also help regarding to my koboldcpp that somehow slows down everytime the program minimized or run before my browser.

I used my koboldcpp with an 8B model alongside with SillyTavern. Everytime I requested for generation from the front, everything went okay including the BLAS processing. But when it comes into the generation phase, it always slows down with an exception if I opened my koboldcpp window, it goes normal again until I minimized it or moved back to my browser. I expected this to be my own performance issue and or idling slowdown issue.

I usually run koboldcpp by disabling mmap, keep foreground, and runs with 8192 tokens for context size (w/ Context Shift).

Can anyone give me at least an explanation for this? I'm looking further for help, thanks!

```
Operating System: Windows 11 Home Single Language 64-bit (10.0, Build 22631) (22621.ni_release.220506-1250)
Processor: 12th Gen Intel(R) Core(TM) i5-12450H (12 CPUs), ~2.0GHz
Memory: 8192MB RAM
Card name: NVIDIA GeForce RTX 3050 6GB Laptop GPU
```

**Comment by LostRuins** [comment, marked as answer]
Follow up here https://github.com/LostRuins/koboldcpp/issues/1180
